### ISSUE TITLE
Better UX experience for deleting multiselect values

### ISSUE THREAD

**Issue by hipertracker** [furqanZafar/react-selectize#104]
Is there any way to customize the selected "tag" value in the multi-select mode so I could just click on it's 'x' small icon to remove it from the selected list? This is how Select2 and react-select work. I think it's more intuitive for new users.

**Comment by furqanZafar** [comment]
you can do that my implemented the renderValue / renderOption props, see here (http://furqanzafar.github.io/react-selectize/#/?category=multi&example=custom-filtering-and-rendering)

**Comment by chiester** [comment]
I don't see renderValue or renderOption used in that example.

**Comment by chiester** [comment]
I came up with a renderValue solution that looks like this: 

  `                    <MultiSelect placeholder="Select..." name="testValues" values={this.state.testValues} options={testValues} onValuesChange={this.myHandler.bind(this)} renderValue={this.renderClosabletags.bind(this)}/>
`                 

`renderClosableTags(item){
        return <div className="simple-value" style={{
            display: "flex",
            alignItems: "center",
            padding: '2px 5px',
            lineHeight: '22px'
        }}>
            <div style={{
                padding: "0 4px",
                color: "rgb(0, 136, 204)"
            }}>
                <i className="fa fa-times" aria-hidden="true"></i>
            </div>
            <div style={{
                marginLeft: 4,
                paddingLeft: 7,
                borderLeft: '1px solid #c9e6f2'
            }}>
                {item.label}
            </div>
        </div>
    }`

Maybe this will be useful to others. Example at http://cdpn.io/BQQdWG

However, there is still one issue. I want to add an onClick event to the div wrapping my "close" icon, but when I do so, it fires when the MultiSelect is rendered and the MultiSelect's own onClick fires when I click on the close icon.

**Comment by chiester** [comment]
It would be useful to have a property in MultiSelect such as dismissableTags that automatically generates tags with the close icon on each tag, as in my example. This is a common feature of tags and it would be nice for it to be generated for users so they don't have to go to the trouble of creating a custom renderValue function, css, etc. 

Consider this a feature request.

**Comment by twig** [comment]
The original selectize js library has simple examples which let you click to select a tag in the multiselect.

https://selectize.github.io/selectize.js/#demo-tagging

Surprised react-selectize doesn't support that bit out of box.

**Comment by furqanZafar** [comment]
@twig tagging & item creation is supported http://furqanzafar.github.io/react-selectize/#/?category=multi&example=tags

**Comment by twig** [comment]
Yeah but you can't select them and then press delete like in the original selectize examples
